src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/dto/InterventionDTO.kt

```diff
@@ -0,0 +1,23 @@
+package uk.gov.justice.digital.hmpps.hmppsinterventionsservice.dto
+
+import uk.gov.justice.digital.hmpps.hmppsinterventionsservice.jpa.entity.Intervention
+import java.time.OffsetDateTime
+import java.util.UUID
+
+data class InterventionDTO(
+  val id: UUID? = null,
```

Comment:
i don't think any of these field should be nullable

Reply:
Yeah, the only issue is that we use InterventionDTO for opencsv to map the csv file to a bean.  So I can set everything apart from id (and possibly created - if we keep that field) to non-null.  The only other option would be to have incoming/outgoing dtos.